composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/tasks/TaskTabContent.kt

```diff
@@ -19,12 +20,22 @@ import org.koin.compose.koinInject
 @Composable
 fun TaskTabContent() {
     val tasksCategoriesState = koinInject<ITaskListState>()
+    val taskListListenerService = koinInject<ITaskListListenerService>()
 
     var isLoading by remember { mutableStateOf(true) }
-
+    var isTaskListEmpty by remember { mutableStateOf(true) }
     LaunchedEffect(Unit) {
         isLoading = true
         tasksCategoriesState.populateTaskListFromServices()
+        if (tasksCategoriesState.isEmpty())
+        {
+            taskListListenerService.startListeningForTaskListThread {
+                isTaskListEmpty = false
+                taskListListenerService.unregisterAllListeners()
+            }
+        } else {
+            isTaskListEmpty = false
+        }
```

Comment:
Jak zostanie dodana nowa lista zadań do taskListState to `taskListState.isEmpty()` powinien zwrócić null.